operators/endpointmetrics/pkg/rendering/renderer.go

```diff
@@ -198,7 +198,7 @@ func Render(
 					s.GetName(),
 				)
 			}
-			// replace the hub alertmanager address
+			// replace the hub alertmanager address. Address will be set to null when alerts are disabled
```

Comment:
is this pr handle the *KS scenario?

Reply:
Yes @marcolan018

Reply:
When alerts are disabled, they are disabled for *all* clusters, not just for OCP. Are there specific concerns @marcolan018 ?

Reply:
I cannot find any related change related with *KS.
if alert forwarding disabled in *KS, just send an empty url to the prometheus?